scripts/push_images.sh

```diff
@@ -101,6 +101,6 @@ for image in "${IMAGES[@]}"; do
     continue
   fi
 
-  build_id=$(cat build.json | jq -r .\[\"depot.build\"\].buildID)
+  build_id=$(cat build.json | jq -r .\[\"depot_{$image}.build\"\].buildID)
```

Comment:
Oh actually I think you want this:

```suggestion
  build_id=$(cat build_${image}.json | jq -r .\[\"depot.build\"\].buildID)
```